app-tracking-protection/vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/store/VpnDatabase.kt

```diff
@@ -137,6 +137,19 @@ abstract class VpnDatabase : RoomDatabase() {
             }
         }
 
+        private val MIGRATION_25_TO_26: Migration = object : Migration(25, 26) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                // Older Android versions do not support column renaming, so we must delete and re-create the table
+                database.execSQL("DROP TABLE IF EXISTS `vpn_app_tracker_exclusion_list`")
+
+                database.execSQL(
+                    "CREATE TABLE IF NOT EXISTS `vpn_app_tracker_exclusion_list` " +
+                        "(`packageName` TEXT PRIMARY KEY NOT NULL, " +
+                        "`reason` TEXT NOT NULL DEFAULT 'UNKNOWN')"
+                )
+            }
+        }
+
```

Comment:
Why do we want to rename to `packageName`. I think `packageId` is not really a misnomer and I'd rather avoid the table drop and the noise of touching so many files just for renaming one field.

Reply:
I was on the fence about this one as well. But our other lists use `packageName` as their field so I though it best if all our lists used either `packageName` or  `packageId` for consistency. This way we can even require this field to be set in our lists in the upcoming privacy-config schemas.

Reply:
Why do we need to rename the db table column to enforce any schema in the JSON?
Our db tables don't (purposely) share schemas with the protocols over the wire.

Also I rather not rename fields to avoid having to think about cases like what happens to users that update the app, but now we drop the table, they need to re-fetch from remote but the eTag hasn't changed.
For example, I haven't tested this PR but I doubt it'd work in all cases, because the `vpn_app_tracker_exclusion_list_metadata` db table–that holds the eTag– is not dropped, and probably is should be as well.

I get the reason to rename, but I don't think is worth the trouble, we `packageId` in plenty of places anyways.

Reply:
Those are great points - I didn't realize the details of the update/eTag mechanism. Let me see if I can re-work this to take in the new name from the schema but not change the DB.